test/test_common.py

```diff
@@ -6,7 +6,7 @@
 _release_versions = [
     54, # last release for GRCh36/hg18
     75, # last release for GRCh37/hg19
-    77, # most recent release for GRCh38
+    78, # most recent release for GRCh38
```

Comment:
Ah, so we're not testing 77 anymore? Even though 78's GTF format is different from 77's? Is that just because 75 is like 77 in terms of GTF formatting?